corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -695,10 +703,14 @@ def create_report(self):
         matching_data_source = self.ds_builder.get_existing_match()
         if matching_data_source:
             data_source_config_id = matching_data_source._id
+            reactivated = False
             if matching_data_source.is_deactivated:
                 matching_data_source.is_deactivated = False
+                reactivated = True
+            changed = self._change_indicators(matching_data_source.configured_indicators)
```

Comment:
This function isn't implemented anywhere: http://manage.dimagi.com/default.asp?218532